services/copyright-service/.env

```
@@ -0,0 +1,51 @@
+PORT=3333
+HOST=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOST key should go before the PORT key

(UnorderedKey)